### DISCUSSION TITLE
Ice server timeout

### DISCUSSION THREAD

**Discussion by gregoriusus** [roman-yerin/vertojs#23] [Q&A] [answered]
Sorry for bothering, but I would really like to test and contribute to this library. 

When I make a call it always waits till iceTimerTriggered is triggered (parameter iceTimeout). onIceGatheringStateChange never gets to complete. Do you have any idea why rtc doesn't complete ice gathering state?

**Comment by roman-yerin** [comment, marked as answer]
This can happen in case you have some network interface that is not connected to internet, a virtual network interface maybe (VirtualBox, VMWare, etc.)
Set iceTimeout to 3-4 seconds. This should be enough in most cases.
